package com.practiceprogram.demo;

public class Test {
	public static void main(String s[]) {

		try {

		int x = 6/0;

		}

		catch(ArithmeticException e1) {

		System.out.println("Arithmetic Exception is thrown");

		}

		}
		}
